src/WorkBC.ElasticSearch.Search/Queries/JobSearchQuery.cs

```diff
@@ -34,6 +34,10 @@ public JobSearchQuery(IGeocodingService geocodingService, IConfiguration configu
             _configuration = configuration;
             _filters = filters;
             _geocodingService = geocodingService;
+            
+            // Apply filter logic 
+            SetSortFilters();
+            SetFilters();
```

Comment:
Move these methods to the constructor.  As far as I can tell, `_filters` is not modified after the class is instantiated.